spyderlib/spyder.py

```diff
@@ -2660,6 +2675,133 @@ def show_tour(self, index):
         self.tour.set_tour(index, frames, self)
         self.tour.start_tour()
 
+    # ---- Check for Spyder Updates
+    def _check_updates_ready(self):
+        """Called by WorkerUpdates when ready"""
+        from distutils.version import StrictVersion as Version

```

Comment:
Instead of using this function, please use our own `check_version` one, present in `utils/programs.py`.